## copied from https://github.com/YDLIDAR/ydlidar_ros2_driver/blob/master/params/ydlidar.yaml
## with modification for TG30
ydlidar_ros2_driver_node:
  ros__parameters:
    port: /dev/ydlidar
    frame_id: laser_frame
    ignore_array: ""
    baudrate: 512000
    lidar_type: 0
    device_type: 0
    sample_rate: 20
    abnormal_check_count: 4
    resolution_fixed: true
    reversion: true
    inverted: true
    auto_reconnect: true
    isSingleChannel: false
    intensity: false
    support_motor_dtr: false
    angle_max: 180.0
    angle_min: -180.0
    range_max: 32.0
    range_min: 0.01
    frequency: 10.0 #8.0
    invalid_range_is_inf: false